inc/aes_packages/namespace.php

```diff
@@ -0,0 +1,318 @@
+<?php
+/**
+ * AES Package File Integration.
```

Comment:
AES seems to be an invented acronym? Assuming this is shows for AWS ElasticSearch? Could we call it `AWS_ES_Packages` at least?

Reply:
Yeah, let's avoid AES, given it's a common acronym for encryption.

Reply:
Ah maybe that's where I picked it up. Sure I'd seen AES meaning Amazon Elasticsearch Service somewhere 🤷